kodi_addon_checker/check_files.py

```diff
@@ -139,3 +140,28 @@ def check_file_permission(report: Report, file_index: list):
         file = os.path.join(file["path"], file["name"])
         if os.path.isfile(file) and os.access(str(file), os.X_OK):
             report.add(Record(PROBLEM, "%s is marked as stand-alone executable" % relative_path(str(file))))
+
+
+def check_print_statement(report: Report, file_index: list):
+    """Check whether any addon files have a print statement in them
+       or not
+        :file_index: list having names and path of all the files present in addon
+    """
+    for file in file_index:
+        if os.path.splitext(file["name"])[1] == '.py':
+            file = os.path.join(file["path"], file["name"])
+
+            with open(file) as f:
```

Comment:
I tested it on windows and this needs an explicit `encoding="UTF-8"`. That's the default for Linux so you may have not noticed any issues.